src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -67,6 +67,14 @@
 _USER_ASSIGNED_IDENTITY = 'userAssignedIdentity'
 
 
+def _load_subscriptions(cli_ctx, all_clouds=False, refresh=False):
```

Comment:
Suggest rename to `load_subscriptions`

Reply:
I agree, and if we do make it public, should it be under the "API" package as well so external teams can formally depend on it?

Reply:
You have a point.